lib/matplotlib/tests/test_ticker.py

```diff
@@ -521,6 +525,15 @@ def test_sublabel(self):
         ax.set_xlim(0.5, 0.9)
         self._sub_labels(ax.xaxis, subs=np.arange(2, 10, dtype=int))
 
+    @pytest.mark.parametrize('val', call_data)
+    def test_LogFormatter_call(self, val):
+        # test _num_to_string method used in __call__
+        fig, ax = plt.subplots()
+        lf = matplotlib.ticker.LogFormatter()
+        ax.xaxis.set_major_formatter(lf)
+
+        assert str(lf(val)) is not None
+
```

Comment:
Didn't realize that there was an option to create a fake axis without having a full plot set up.